File: GrowthWare_VB/GrowthWareFramework/ModelObjects/MNotificationsProfile.vb
Imports GrowthWare.Framework.Enumerations
Imports GrowthWare.Framework.ModelObjects.Base

Namespace ModelObjects
    <Serializable(), CLSCompliant(True)> _
    Public Class MNotificationsProfile

#Region "Member Properties"
        Private m_SE_SEQ_ID As Integer = -1
        Private m_Status As Integer = 1
        Private m_Account As String = String.Empty
        Private m_Function_Seq_ID As Integer = -1
#End Region

#Region "Public Properties"
        Public Property SecurityEntityID() As Integer
            Get
                Return m_SE_SEQ_ID
            End Get
            Set(ByVal value As Integer)
                m_SE_SEQ_ID = value
            End Set
        End Property

        Public Property Status() As Boolean
            Get
                Return m_Status
            End Get
            Set(ByVal value As Boolean)
                m_Status = value
            End Set
        End Property

        Public Property Account() As String
            Get
                Return m_Account
            End Get
            Set(ByVal value As String)
                m_Account = value.Trim
            End Set
        End Property

        Public Property FunctionID() As Integer
            Get
                Return m_Function_Seq_ID
            End Get
            Set(ByVal value As Integer)
                m_Function_Seq_ID = value
            End Set
        End Property
#End Region

#Region "Public Methods"
            ''' <summary>
            ''' Provides a new account profile with the default vaules
            ''' </summary>
            ''' <remarks></remarks>
        Public Sub New()

        End Sub
#End Region
    End Class
End Namespace

File: GrowthWare_CS/GrowthWareFrameworkModel/Profiles/MNotificationsProfile.cs

using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Diagnostics;
using System.Web;
using GrowthWare.Framework.Enumerations;
using GrowthWare.Framework.ModelObjects.Base;

namespace GrowthWare.Framework.ModelObjects
{
	[Serializable(), CLSCompliant(true)]
	public class MNotificationsProfile
	{

#region "Member Properties"
		private int m_SE_SEQ_ID = -1;
		private bool m_Status = true;
		private string m_Account = string.Empty;
		private int m_Function_Seq_ID = -1;
#endregion

#region "Public Properties"
		public int SecurityEntityID {
			get { return m_SE_SEQ_ID; }
			set { m_SE_SEQ_ID = value; }
		}

		public bool Status {
			get { return m_Status; }
			set { m_Status = value; }
		}

		public string Account {
			get { return m_Account; }
			set { m_Account = value.Trim(); }
		}

		public int FunctionID {
			get { return m_Function_Seq_ID; }
			set { m_Function_Seq_ID = value; }
		}
#endregion

#region "Public Methods"
		/// <summary>
		/// Provides a new account profile with the default vaules
		/// </summary>
		/// <remarks></remarks>

		public MNotificationsProfile()
		{
		}
#endregion
	}
}
